`create_table` with a PyArrow Schema (#305)
apache/iceberg-python: mkdocs/docs/api.md

)
```

> One can also create an Iceberg table using a pyarrow schema:

Reviewer: nit: 
```suggestion
To create a table using a pyarrow schema:
```

```python
import pyarrow as pa

> pa.schema(
>     [
>         pa.field("foo", pa.string(), nullable=True),
>         pa.field("bar", pa.int32(), nullable=False),
>         pa.field("baz", pa.bool_(), nullable=True),
>     ]
> )
> 
> catalog.create_table(
>     identifier="docs_example.bids",
>     schema=schema,
>     location="s3://pyiceberg",
> )

Reviewer: ```suggestion
schema = pa.schema(
    [
        pa.field("foo", pa.string(), nullable=True),
        pa.field("bar", pa.int32(), nullable=False),
        pa.field("baz", pa.bool_(), nullable=True),
    ]
)

catalog.create_table(
    identifier="docs_example.bids",
    schema=schema,
)
```
